torch/_inductor/codegen/triton.py

```diff
@@ -2544,7 +2570,7 @@ def _mask_value(value, default) -> CSEVariable:
             exit_stack = contextlib.ExitStack()
             for buf in (self.post_loop_combine, self.post_loop_store):
                 # only do cooperative reduction combines if we have more than one thread block
-                buf.writeline("if RSPLIT > 1:")
+                buf.writeline("if HAS_RSPLIT:")
```

Comment:
just curious - was there a triton reason you had to do this or readability ?

Reply:
For readability